### ISSUE TITLE
Package not compatible with PHP 7.3

### ISSUE THREAD

**Issue by RickyLam11** [laravel-notification-channels/google-chat#4]
I am trying to send out a simple message

in config/google-chat.php
```<?php

return [
    'space' => env('GOOGLE_CHAT_DEFAULT_SPACE', 'https://chat.googleapis.com/v1/spaces/XXX/messages?key=YYY&token=ZZZ'),

    'spaces' => [
        'testing' => 'https://chat.googleapis.com/v1/spaces/XXX/messages?key=YYY&token=ZZZ',
    ],
];
```

in my test controller
```
<?php
...
use NotificationChannels\GoogleChat\GoogleChatMessage;

class TestController extends Controller
{
    public function index()
    {
        GoogleChatMessage::create('An invoice was paid!')->to('testing');
    }
}

```

I get a ParseError from Laravel
```
syntax error, unexpected 'array' (T_ARRAY), expecting function (T_FUNCTION) or const (T_CONST)
```
![image](https://user-images.githubusercontent.com/29765078/124908819-5255e880-e01c-11eb-9d2c-ed3196b94f61.png)

I tried `composer dumpautoload` but still face the same error.

**Comment by RickyLam11** [comment]
It turns out that type hint in class variables cause the problem. 

In GoogleChatMessage.php

```
<?php
...
    /**
     * The configured message payload.
     *
     * @var array
     */
    protected array $payload = [];

    /**
     * The Space's webhook URL where this message should be sent.
     *
     * @var string|null
     */
    protected ?string $endpoint = null;
```

The type hint `array` / `?string` trigger the error.
Same issue happen in all class in /src.
I can send out Card after I remove them.

Please help take a look.

PS. I checked [Microsoft Teams](https://github.com/laravel-notification-channels/microsoft-teams/blob/master/src/MicrosoftTeams.php), there is no type hint in class variables.

**Comment by Mamoy23** [comment]
Same error.

**Comment by frankieeedeee** [comment]
Hey @RickyLam11 and @Mamoy23 - Thanks for flagging this, appreciate you taking the time to make an issue 🙂 Looks like this is a compatibility error with PHP 7.3. I initially developed this with the intent on support >=7.4 as 7.3 is no longer officially supported, however 7.3 is defined as the minimum PHP requirement in composer. So this is an oversight on my part, apologies!

Since 1.x is now being actively used, 7.3 will have to be supported. We'll need to:
- [ ] Remove property type hints
- [ ] Update GH actions test matrix to include 7.3

Will work on this ASAP unless anyone else beats me to it! Cheers

**Comment by RickyLam11** [comment]
Good to know ~

Please raise if I can help ~~

**Comment by frankieeedeee** [comment]
@RickyLam11 Do you want to try using `composer require laravel-notification-channels/google-chat:dev-bug-php73`? This is the branch I'm about to merge :)

**Comment by frankieeedeee** [comment]
Resolved in v1.0.1 🎉

Cc: @RickyLam11 @Mamoy23

**Comment by RickyLam11** [comment]
@frankieeedeee yes v1.0.1 is working. Thanks ~
